lib/ruby_pocket/cli/open_action.rb

```diff
@@ -1,11 +1,25 @@
 module RubyPocket
   module Cli
     class OpenAction
+      OPEN_COMMANDS = {
+        linux: 'xdg-open',
+        mac: 'open',
+        windows: 'start'
+      }.freeze
+
       def call(options)
         favorite = Favorite[options.values[:id]]
-        fail ArgumentError, 'Favorite not found!' unless favorite
+        raise ArgumentError, 'Favorite not found!' unless favorite
+
+        open_url(favorite.url)
+      end
+
+      private
+
+      def open_url(favorite_url)
+        command = OPEN_COMMANDS.fetch(OS.os, 'open')
```

Comment:
Should the fallback be Unix? Can we safely assume `xdg-open` in that case?

If dealing with an unknown OS, what do you think about `abort "Operating system not supported!"`?

Reply:
From what I've researched, `xdg-open` will be available for the majority of the time. It is part of freedesktop, and it so it should be available in most desktop environments. As for OSs without a desktop environment, I don't think it would work, but I couldn't find a more generic command for Linux distros.

Reply:
Just added an abort when OS is unknown